samples/boards/nrf52/mesh/onoff_level_vnd_app/src/common.c

```diff
@@ -0,0 +1,124 @@
+/*  Bluetooth: Mesh Generic OnOff, Generic Level & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant
```

Comment:
Could you please use your full name here, so that it matches the git author and signed-off-by. Also, are you sure that the copyright belongs to you and not your company? Most of the time it's the employer that owns the work-related code that's written by an employee.

Reply:
One other thing: it'd be appropriate to also have a second copyright line from the onoff-app for any files where you've derived code from that app.

Reply:
Actually, just forget about this for now - I just realized that the author (Steve Brown) didn't put the right copyright to begin with. So just leave it with your own line.